webpack/make-webpack-config.js

```diff
@@ -0,0 +1,191 @@
+var packageInfo = require('../package');
+var path = require('path');
+/*
+  Plugins
+ */
+var autoprefixer = require('autoprefixer');
+var colorLighten = require('less-color-lighten');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var StringReplacePlugin = require('string-replace-webpack-plugin');
+
+var analyticsKey = '39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP';
+
+var VARS = {
+  VERSION: packageInfo.version,
+  ENV: process.env.NODE_ENV,
+  ANALYTICS_KEY: analyticsKey
+}
+
+function absPath() {
+  var args = [].slice.apply(arguments);
+  args.unshift(__dirname, '..');
+
+  return path.resolve.apply(path.resolve, args);
+}
+
+function extractForProduction(loaders) {
+  return ExtractTextPlugin.extract('style', loaders.substr(loaders.indexOf('!')));
+}
+
+module.exports = function (options) {
+  var cssLoaders = 'style!css!postcss';
+  var scssLoaders = cssLoaders + '!sass';
+  var sassLoaders = scssLoaders + '?indentedSyntax=sass';
+  var lessLoaders = cssLoaders + '!less';
```

Comment:
If we want to provide this freedom of choice, we should consider different plugin mechanisms. They're so tightly coupled at the moment that they should use the very same tools to reduce the amount of conflicts due to different tooling.